src/Controller/CalculatorController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Order;
+use App\Form\CalculatorType;
+use DateTime;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/', name: 'calculator_')]
+class CalculatorController extends AbstractController
+{
+    #[Route('/', name: 'index')]
+    public function index(Request $request): Response
```

Comment:
what is the difference between these 2 methods?

Reply:
as we discussed, i've modified the controller to let the client handle calculations before submitting